src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -690,8 +695,14 @@ public void onUpdate() {
                 if (sync) {
                     sync(syncPacket, syncToAll);
                 }
-            } else {
 
+                if (victim != -1 && biteTickCounter >= 20) {
```

Comment:
Just a tiny improvement, but we only need to increase the biteTickCounter if there is a victim.
e.g. with something like
`if(victim !=-1 && biteTickCounter++ >= 20){`